nucypher/network/server.py

```diff
@@ -189,14 +192,12 @@ def node_metadata_exchange():
             payload = this_node.known_nodes.snapshot() + bytes(FLEET_STATES_MATCH)
             signature = this_node.stamp(payload)
             return Response(bytes(signature) + payload, headers=headers)
-
-        sprouts = _node_class.batch_from_bytes(request.data)
-
-        for node in sprouts:
-            this_node.remember_node(node)
-
-        # TODO: What's the right status code here?  202?  Different if we already knew about the node(s)?
-        return all_known_nodes()
+        else:
+            sprouts = _node_class.batch_from_bytes(request.data)
+            for node in sprouts:
+                this_node.remember_node(node)
+            # TODO: What's the right status code here?  202?  Different if we already knew about the node(s)?
```

Comment:
This TODO looks completely out of place. Did it somehow migrate from `server.py`?